src/js/count.js

```diff
@@ -8,6 +8,8 @@ class Count {
 	constructor (countEl, opts) {
 		this.countEl = countEl;
 		this.articleId = opts.articleId;
+		this.useStagingEnvironment = opts &&
+			(opts.useStagingEnvironment === true || opts.useStagingEnvironment === 'true');
```

Comment:
Do you think we need to check if it's `true` or `'true'`? I think we could possibly just check if its truthy

```suggestion
			(!!opts.useStagingEnvironment);
```

Reply:
What if there is a data attribute like this: `data-o-comments-use-staging-environment="false"`?

Reply:
Discovered that the code that converts data attributes to options converts string "true" to boolean `true`. So doing the suggested change is perfectly safe.